v0/destinations/am/transform.js

```diff
@@ -622,7 +623,8 @@ function process(event) {
   const messageType = message.type.toLowerCase();
   const toSendEvents = [];
   if (messageType === EventType.TRACK) {
-    if (message.properties && message.properties.revenue) {
+    const { properties } = message;
+    if (properties && isDefinedAndNotNull(properties.revenue)) {
```

Comment:
Please add new test cases to cover this bug scenario

Reply:
Added test case